mscore/alsa.cpp

```diff
@@ -667,7 +667,7 @@ void AlsaAudio::alsaLoop()
       rt_param.sched_priority = 50;
       int rv = pthread_setschedparam(pthread_self(), SCHED_FIFO, &rt_param);
       if (rv == -1)
-            perror("MuseScore: set realtime scheduler failed");
+            perror("Set realtime scheduler failed");
```

Comment:
that is an interesting one. Does it make any sense to continue after this error? Shouldn't it be a dialog rather than something a Windows user would never get to see?